README.rst

```diff
@@ -86,7 +98,7 @@ From the git directory run the following commands::
     pip install sphinx
     sphinx-build threedi-docs\source build
 
-You will recieve some warnings, check them. You are not able to evaluate LateX
+You will receive some warnings; check them. You are not able to evaluate LateX
```

Comment:
Ooit hoorde ik een praatje van iemand die automatisch de spelling in documentatie liet controleren:
https://reinout.vanrees.org/weblog/2018/05/24/08-spell-checking.html

Misschien iets om een keer een blik op te werpen. Want foutjes zoals `recieve` zullen bijvoorbeeld vaker voorkomen.